prbt_hardware_support/include/prbt_hardware_support/modbus_adapter_brake_test.h

```diff
@@ -19,28 +19,35 @@
 #define MODBUS_ADAPTER_BRAKE_TEST_H
 
 #include <memory>
+#include <map>
+#include <string>
+#include <functional>
 
 #include <ros/ros.h>
 
-#include <prbt_hardware_support/adapter_brake_test.h>
 #include <prbt_hardware_support/ModbusMsgInStamped.h>
 #include <prbt_hardware_support/modbus_api_spec.h>
-#include <prbt_hardware_support/filter_pipeline.h>
+#include <prbt_hardware_support/IsBrakeTestRequired.h>
+#include <prbt_hardware_support/SendBrakeTestResult.h>
 
 namespace prbt_hardware_support
 {
 
+using TRegVec = std::vector<uint16_t>;
```

Comment:
I am in favor of dropping the "T" prefix since it does not provide useful information. For me this is something like a bit hungarian notation. https://en.wikipedia.org/wiki/Hungarian_notation